templates/body.hbs

```diff
@@ -5,12 +5,12 @@
     <div id="operations">
         <button type="button" class="btn btn-primary btn-sm" onclick="htmlapi.up()" {{#unless operations.up}}disabled="disabled"{{/unless}}><span class="glyphicon glyphicon-arrow-up"></span> Up</button>
         <button type="button" class="btn btn-primary btn-sm" onclick="htmlapi.reload()"><span class="glyphicon glyphicon-refresh"></span> Reload</button>
-      {{#if operations.post}}
+      {{!-- {{#if operations.post}}
```

Comment:
I decided to skip comment out instead of deleting to make this easier to find in the future and this repo isn't anything like the code we typically work on.